docs/reference/regions.md

```diff
@@ -0,0 +1,22 @@
+---
+id: regions
+title: "Regions"
+description: "After creating a cluster, specify a region for that cluster. Read on for details of Google Cloud Platform regions currently supported in Camunda Platform 8 SaaS."
+---
+
+When you create a cluster in Camunda Platform 8 SaaS, you must specify a region for that cluster.
+
+Below, find a list of regions currently supported in Camunda Platform 8 SaaS.
+
+Currently, we only make these regions available for customers on the Enterprise Plan. These customers can discuss regions with their Customer Success Manager. Professional and Trial users are currently hosted on US Central.
```

Comment:
Would it be worth linking to details on the [Enterprise Plan] here for those interested?

Reply:
Good question, I wasn't able to find a good link on Enterprise Plan details, but I was able to find a separate contact us page. I've asked internally if there is a difference and whether we should include this - https://camunda.slack.com/archives/C01UMP0V17Z/p1672937305910939